eslint-plugin/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "eslint-plugin-tsdoc",
+  "version": "0.1.0",
+  "description": "eslint plugin for tsdoc",
+  "private": false,
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc",
```

Comment:
We probably want to run `eslint` on this package as well. Maybe copy parts of [the build script](https://github.com/microsoft/tsdoc/blob/master/tsdoc/build.js) from `tsdoc`.